.gitignore

```diff
@@ -8,6 +8,7 @@ assets/hashes.js
 backend_prod_files/*
 local_compiled_js/*
 core/templates/prod/*
+core/templates/dev/head/dist/*
```

Comment:
Okay, it will take a little bit of time, because of the differences between dev and prod, that are right now mitigated by the fact that the folder is in the `templates/dev/head`.

Reply:
Now that I think of it, we should keep it in this folder for now and move it later in other PRs later. Although, it is not ideal place for the folder, moving it outside would require more modifications to the build process and since we need to merge this as soon as possible I think it is better to keep it here now. I can probably rename the folder to some better suiting name.

Reply:
SG, let's file an issue for it.